docs/b2c-distance-selling.md

```diff
@@ -0,0 +1,13 @@
+## Distance Selling (B2C)
+
+<img src="images/distance-selling-b2c.png" alt="Distance Selling (B2C)">
+
+This part of the plugin handles taxes for Digital Selling for Consumers. As a store owner, you will need to register for EU Tax ID in countries where you reach [Distance Selling EU Tax thresholds](https://www.vatlive.com/eu-vat-rules/distance-selling/distance-selling-eu-vat-thresholds/).
+
+Currently, the plugin offers 2 options for this class of taxes.
+
+- **EU VAT Handling for Distance Selling** - Enabling this option will allow you to collect taxes on transactions done via countries which fall under the "Distance Selling" category. You will be responsible for specifying the countries for this category in the next option. Only the countries added will be applicable for tax calculation at the time of checkout. Also, this option has no impact on the B2B transactions.
+
+- **Select countries for which you would like to import tax rates** - Add countries to this multi-select option box whose tax rates you would like to import. As of now, the plugin supports only the EU countries.
```

Comment:
"As of now, the plugin supports only the EU countries." - remove this. The whole tax is related only to EU countries.